pkg/crc/api/api_http_test.go

```diff
@@ -126,6 +126,33 @@ func TestHTTPApi(t *testing.T) {
 		},
 		configSetResult,
 	)
+	// Get the cpus config again after setting it to make sure it is set
+	// properly.
+	configGetAfterSetResult, err := client.GetConfig([]string{"cpus"})
```

Comment:
Can we also add a test case for the case of getting all the configs so that we don't have regressions in the future. This was also asked in https://github.com/code-ready/crc/pull/2631#issuecomment-894075772

Reply:
@anjannath Added 👍🏼